packages/client/tests/e2e/engine-not-found-error/bundler-tampered-with-engine-copy/package.json

```diff
@@ -8,7 +8,6 @@
   },
   "devDependencies": {
     "@types/node": "18.19.76",
-    "esbuild": "0.21.4",
```

Comment:
btw, here this works, unlike wrangler, because esbuild is globally installed in the docker image